Source/Noesis.Javascript/JavascriptInterop.cpp

```diff
@@ -391,7 +391,7 @@ JavascriptInterop::ConvertObjectFromV8(Local<Object> iObject, ConvertedObjects &
  * and assumes incorrectly that Germany observed UTC+2 during the summer time.
  *
  * V8
- * new Date(1978, 5, 15) // "Thu Jun 15 1978 00:00:00 GMT+0100 (Mitteleurop�ische Normalzeit)"
+ * new Date(1978, 5, 15) // "Thu Jun 15 1978 00:00:00 GMT+0100 (Mitteleurop�ische Normalzeit)"

```

Comment:
Either the encoding of the file got messed up here, or it was locally messed up and then committed with a replacement character.